.circleci/config.yml

```diff
@@ -40,6 +42,52 @@ jobs:
         paths:
           - .
 
+
+  run-integration-tests:
+    machine:
+      image: circleci/classic:201808-01
+    environment:
+      K8S_VERSION: v1.16.8
+      KUBECONFIG: /home/circleci/.kube/config
+      MINIKUBE_VERSION: v1.13.0
+      MINIKUBE_WANTUPDATENOTIFICATION: false
+      MINIKUBE_WANTREPORTERRORPROMPT: false
+      MINIKUBE_HOME: /home/circleci
+      CHANGE_MINIKUBE_NONE_USER: true
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: setup kubectl
+          command: |
+            curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+            mkdir -p ${HOME}/.kube
+            touch ${HOME}/.kube/config
+      - run:
+          name: setup minikube
+          command: |
+            curl -Lo minikube https://github.com/kubernetes/minikube/releases/download/${MINIKUBE_VERSION}/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+      - run:
+          name: setup helm
+          command: curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash
+      - run:
+          name: Download kustomizable
+          command: |
+            KUSTOMIZE_VERSION=2.1.0
+            sudo chown -R $(whoami) /usr/local/bin
+            curl -L -o /usr/local/bin/kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64
+            chmod +x /usr/local/bin/kustomize
+            kustomize version
+      - run:
+          name: Run Integration Test
+          command:  |
+            make gen-yaml
+            cd tests
+            export IS_LOCAL=false
+            ./run.sh "1.16.8" "1.5.7" "../out"
```

Comment:
I think we should assert to make sure the tests are passing, either we can exit with status 1 in test1.sh and test2.sh or grep for PASS from the output of `./run.sh "1.16.8" "1.5.7" "../out"`. Ex: `./run.sh "1.16.8" "1.5.7" "../out" | grep "PASS" | wc -l` will return 2 since there are 2 test cases.

Also, we should separate out each combination as a separate run command. For example, we should have - run: name: `Run Integration Test for Istio 1.5.7`. That way we can see which combination failed and which passed.

Reply:
this make sense, I'm working on it